### ISSUE TITLE
No right-click menu

### ISSUE THREAD

**Issue by Jayman2000** [radicle-dev/radicle-upstream#2733]
When I right-click on text, I expect a menu to open giving me the option to copy the text. Upstream doesn’t have one.

**Comment by rudolfs** [comment]
You're right, we don't have a context menu on right-click. But we do have the `CopyableIdentifier` component. Would that work?

**Comment by Jayman2000** [comment]
> You're right, we don't have a context menu on right-click. But we do have the `CopyableIdentifier` component. Would that work?

I don’t think so. I’m not exactly sure what `CopyableIdentifier`s are, but I’m assuming that they’re like one of these:

![When you hover over a project’s Radicle ID, a small bubble appears that says “Copy Radicle ID to clipboard”.](https://user-images.githubusercontent.com/5579359/150393176-96ffbda5-4f4b-4774-b909-7aaac1b2c0d9.png)

Here’s the situation I was in when I filed this issue. I was proofreading a Markdown file in Upstream, and I noticed that a link didn’t work. I was wondering whether there was a problem with the URL or if it was just a symptom of #2466. I had the Markdown file open in both Upstream and a plain text editor. I wanted to copy the link’s text and search for it in the text editor to quickly get to the portion of the document that I was looking for.

When I highlight text using the mouse, I find it natural to use the right click menu to copy. That being said, I can still use <kbd><kbd>Ctrl</kbd>+<kbd>C</kbd></kbd>, so it’s not that big of a deal.
